imagetest/test_suites/mdsmtls/mds_mtls_test.go

```diff
@@ -72,9 +86,61 @@ func checkCredsPresentWindows(t *testing.T) {
 }
 
 func TestMTLSCredsExists(t *testing.T) {
+	checkMTLSAvailable(t)
+	ctx := utils.Context(t)
+	var rootKeyFile, clientKeyFile string
 	if utils.IsWindows() {
+		rootKeyFile = filepath.Join(os.Getenv("ProgramData"), "Google", "Compute Engine", "mds-mtls-root.crt")
+		clientKeyFile = filepath.Join(os.Getenv("ProgramData"), "Google", "Compute Engine", "mds-mtls-client.crt")
```

Comment:
nit: its `mds-mtls-client.key` here and `client.key` on line 98

Reply:
Done